.travis.yml

```diff
@@ -0,0 +1,32 @@
+# Copyright 2019 Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: rust
+rust:
+  - stable
+  - beta
+  - nightly
+matrix:
+  allow_failures:
+    - rust: nightly
+  fast_finish: true
+install:
+  - rustup component add rustfmt
+  - cargo install --force cargo-audit
+  - cargo generate-lockfile
+script:
+  - cargo fmt -- --check
+  - git rev-list origin/master.. | xargs -n1 -I{} sh -c '! git rev-parse {}^2'
```

Comment:
This line is currently failing on CI. I'm not sure how to fix it though. What's the purpose of it?

Reply:
Merge commit detection. It was working in GH Actions, but I'll need to tweak it to make it work in Travis.